src/data-dictionary/events/K8sDaemonsetSample/label.LABEL_NAME.md

```diff
@@ -0,0 +1,9 @@
+---
+name: label.LABEL_NAME:w
```

Comment:
typo with the 'w'. I'm also thinking maybe we make it clearer when we have an attribute value that includes customer-specific data. Like maybe 'YOUR_LABEL_NAME' to make it very clear.

Reply:
Yes, this was my mistake! It's fixed now!